plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoClientConfig.java

```diff
@@ -53,6 +53,7 @@ public void testDefaults()
     public void testExplicitPropertyMappings()
     {
         Map<String, String> properties = new ImmutableMap.Builder<String, String>()
+                .put("mongodb.connection-url", "")
```

Comment:
Move to before `mongodb.credentials` and avoid using empty string as the value. e.g. `mongodb://mongodb0.example.com:27017`

Reply:
It has to be empty because seeds and connection-url cannot both have a value and it cannot be null because thats the default value